src/ORM/Connect/PDOConnector.php

```diff
@@ -483,7 +483,7 @@ public function getLastError()
 
     public function getGeneratedID($table)
     {
-        return $this->pdoConnection->lastInsertId();
+        return intval($this->pdoConnection->lastInsertId());
```

Comment:
This is a stylistic thing which may be bordering on opinionated, but I'd probably prefer `(int) $this...` casting over using `intval`. I think that's usually how we cast in other parts of core code.

Reply:
I created a second PR (#8874) that uses (int) to do the casting. PSR-2 actually states that (int) should be preferred to intval(). What I could not find is a statement wether there should be a space after type cast. So I just went with your sample code.